extension/src/vscode/inputBox.ts

```diff
@@ -1,37 +1,58 @@
 import { window } from 'vscode'
 import { Title } from './title'
 import { isValidStringInteger } from '../util/number'
+import { getIsoDate, isFreeTextDate } from '../util/date'
 
-export const getInput = (title: Title, value?: string) =>
+export const getInput = (
+  title: Title,
+  value?: string
+): Thenable<string | undefined> =>
   window.showInputBox({
     title,
     value
   })
 
-export const getValidInput = (
+const getValidInput = (
   title: Title,
   validateInput: (text?: string) => null | string,
   options?: { prompt?: string; value?: string }
-) =>
+): Thenable<string | undefined> =>
   window.showInputBox({
     prompt: options?.prompt,
     title,
     validateInput,
     value: options?.value
   })
 
+const isValid = (
```

Comment:
Should this function be `isValidNumber` since the function is focused on checking numbers?